### ISSUE TITLE
TestCase failing - t/register.t in v2.16

### ISSUE THREAD

**Issue by sudo-batman** [jhthorsen/mojolicious-plugin-openapi#145]
We recently started bundling our Mojolicious App in Docker,
In my Docker file I am running this command -
`RUN curl -L cpanmin.us | perl - Mojolicious::Plugin::OpenAPI@2.16`
to install the latest version
While installing v2.16 I am getting following error in testcases-

```
#   Failed test 'exact match for JSON Pointer "/responses/400/content/application~1json/schema/$ref"'
#   at t/register.t line 89.
#          got: '#/definitions/_components_responses_DefaultResponse'
#     expected: '#/definitions/__components_responses_DefaultResponse'
# Looks like you failed 1 test of 39.
t/register.t ..........................
Dubious, test returned 1 (wstat 256, 0x100)
Failed 1/39 subtests
```

I tried to install older version v2.14 and it install successfully. 
It is giving the similar error for v 2.15.
From change logs it looks like open-api v3.0 was introduced in v2.15.
Is there any extra dependency introduced in v2.15 which may be missing on my side or is this a genuine issue ?

OS - Ubuntu:18.04
Perl - 5.26
Mojolicious - 8.02

Let me know in case you want more details?

**Comment by HEM42** [comment]
I noticed this as well, and created a PR #143 yesterday.

I should properly have made an issue as well

**Comment by sudo-batman** [comment]
Thank you

**Comment by jhthorsen** [comment]
Closed by #143.
